backend/api/models/Badge.py

```diff
@@ -0,0 +1,87 @@
+import pdb
+
+# this file and the classes in it do not correspond to a table in the Database
+
+
+def num_quiz_attempts(user):
+    return len(user.attempted_quizzes)
+
+
+class Badge:
+    def __init__(self, id, title, description, congrats_text, graphic):
+        self.id = id
+        self.title = title
+        self.description = description
+        self.congrats_text = congrats_text
+        self.graphic = graphic
+
+    def can_receive(self, user, quiz_result):
```

Comment:
quick question: does the can_receive function receive a user object and the quiz_result that is about to be added to the user's attempted quizzes? Or does it receive the user with the quiz_result added?